README.md

```diff
@@ -29,7 +29,8 @@ Or install it yourself as:
 | Abucoins          | Y       | Y          | Y       |         | Y           |          | abucoins          |
 | ACX               | Y       | Y          | Y       |         | Y           | Y        | acx               |
 | AEX               | Y       | Y          | Y       |         | Y           |          | aex               |
-| Allcoin           | Y       | Y          | Y       |         | Y           |          | allcoin           |
+| Airswap           | Y       |            |         |         | Y           |          | airswap           |
+| Allcoin           | Y       | Y          |         |         | User-Defined|          | allcoin           |
```

Comment:
please discard non-related change. Rebase your branch against master

Reply:
erm I already rebase onto master, am I right? here I add the airswap to readme, so it considers related?

Reply:
Allcoin should have no changes.

Reply:
oh I see